kahm/values.yaml

```
@@ -116,6 +116,9 @@ postgresql-ha:
     podAntiAffinityPreset: hard
     replicaCount: 3
     podLabels:
+      release: "{{ .Release.Name }}"
+      app.kubernetes.io/namespace: "{{ .Release.Namespace }}"
+      objectscale.dellemc.com/logging-inject: "true"
```

this should be set to: 

```suggestion
      release: "{{ .Release.Name }}"
{{ include "common-lib.logging-inject-common-labels" . | indent 8 }}
```